conans/client/rest/conan_requester.py

```diff
@@ -55,6 +57,9 @@ def _add_kwargs(self, url, kwargs):
                 kwargs["proxies"] = self.proxies
         if self._timeout_seconds:
             kwargs["timeout"] = self._timeout_seconds
+        if not kwargs.get("headers"):
+            kwargs["headers"]  = {}
+        kwargs["headers"]["User-Agent"] = "Conan/%s (Python %s) %s" % (client_version,platform.python_version(),requests.utils.default_user_agent())
```

Comment:
Almost there! Now, I think the line is more than 100 chars, and after the comma it should be an space (or ln), you can split it after the comma:

```python
kwargs["headers"]["User-Agent"] = "Conan/%s (Python %s) %s" % (client_version, 
                                                               platform.python_version(), 
                                                               requests.utils.default_user_agent())
```